test/utils/UtilsSpec.scala

```diff
@@ -57,4 +60,49 @@ class UtilsSpec extends SpecBase {
       referrerUrl(None) mustBe Option(s"$path")
     }
   }
+
+  "isEqualOrAfter" must {
+    "return true when date is equal to or after cutOffDate" in new Setup {
+      Utils.isEqualOrAfter(date03, cutOffDate) shouldBe true
```

Comment:
Could you please remove Utils. to access the methods as we are already in Utils test class?  Same for other lines